util/src/main/java/org/datacommons/util/SummaryReportGenerator.java

```diff
@@ -82,8 +85,13 @@ public static void generateReportSummary(
     Template template = cfg.getTemplate("SummaryReport.ftl");
     HashMap<String, Object> data = new HashMap<>();
     data.put("levelSummary", log.getLevelSummaryMap());
-    data.put("svSummaryMap", svSummaryMap);
     data.put("commandArgs", log.getCommandArgs());
+    if (TEST_mode) {
+      svSummaryMap = new TreeMap<>(svSummaryMap);
+      placeSeriesSummaryMap = new TreeMap<>(placeSeriesSummaryMap);
```

Comment:
comment?

(is this for ordering tests?)

Reply:
done